docs/preparing-your-app-for-testflight.md

```diff
@@ -0,0 +1,143 @@
+
+There is some preparation involved with preparing your Apple Developer account for publishing, such as preparing an App Identifier and a Provisioning Profile. The MAUI documentation has a great guide for that:
+https://aka.ms/maui-publish-app-store
+
+Also, your project will need to have configuration values set in the .csproj file, the Info.plist file, and the Entitlements.plist file.
+
+## Preparing your Project File (.csproj) for App Store submission
+
+Before you can submit your app to the App Store, you need to make sure that it is correctly configured. This includes:
+
+- Setting the build to reference the correct `Entitlements.plist` in your project (For example, one for Debug and one for Release).
+- Setting the correct `ApplicationId` in your project.
+- Setting the correct `ApplicationVersion` in your project.
+- Setting the correct `CodesignKey` in your project.
+- Setting the correct `PackagingSigningKey` in your project.
+- Setting the correct `CodesignEntitlements` in your project.
+- Setting the correct `CodesignProvision` in your project.
+
+### Info.plist
+
+The `Info.plist` file is used by the build system to configure your app and is already generated by your project. It contains info such as the Bundle Display Name and Bundle Identifier.
+### Entitlements.plist
+
+The `Entitlements.plist` file is used by the build system to sign your app.
+
+Some of the values in the `Entitlements.plist` file are set by the build system. These values are:
+
+- com.apple.security.app-sandbox
+- com.apple.security.network.client
+
+## Building your App for Publishing
+
+Once you've set all the correct values, you can build your app for publishing. You can build your app for publishing very easily by using the dotnet build command. Here is an example for building a MacCatalyst app with .NET 7.0:
+
+`dotnet build -f:net7.0-maccatalyst -c:Release`
+
+## App Store Connect
+
+App Store Connect is the web site that you use to manage your apps on the App Store. You can use App Store Connect to upload your app to the App Store, and you can use App Store Connect to manage your app once it is on the App Store.
+
+### Creating an App in App Store Connect
+
+Before you can upload your app to the App Store, you need to create an app on App Store Connect. You can create an app on App Store Connect by following these steps:
+
+1. Go to [https://appstoreconnect.apple.com](https://appstoreconnect.apple.com).
+2. Click on "My Apps".
+3. Click on the "+" button.
+4. Enter the name of your app.
+5. Select the platform for your app.
+6. Select the primary language for your app.
+7. Select the category for your app. Note: It is important that the primary category you select here will be the one 
+8. Click on "Create".
+
+### Uploading your App using Transporter
+
+Once you have created an app on App Store Connect, you can upload your app to the App Store. You can upload your app to the App Store by following these steps:
```

Comment:
```suggestion
Once you have created an app on App Store Connect, you can upload your app to the App Store by following these steps:
```

Reply:
Just a suggestion for conciseness - can be applied to the below examples if wanted